Prototype/PR-hub Setup.md

```diff
@@ -0,0 +1,29 @@
+# Prototype Hub setup
+```bash
+# Create connect to prototype netværk script
+echo "nmcli device wifi conenct xpowertest hidden yes password xpowertest ifname wlan0" > ~/connect2test && chmod u+x connect2test
```

Comment:
* conenct -> connect
* ssid should be with "" in last test so it becomes "xpowertest"

Reply:
resolved